_stbt/frameobject.py

```diff
@@ -93,6 +122,7 @@ def __init__(cls, name, parents, dct):
         super(_FrameObjectMeta, cls).__init__(name, parents, dct)
 
 
+@for_object_repository()
```

Comment:
A bit late but is this decorator appropriate here? You only need it on indirect **subclasses** of FrameObject.

Reply:
Raised #635.